x-pack/plugins/search_playground/server/routes.ts

```diff
@@ -75,6 +75,8 @@ export function defineRoutes({ log, router }: { log: Logger; router: IRouter })
               log.error('Failed to parse the Elasticsearch query', e);
             }
           },
+          content_field: JSON.parse(data.source_fields),
```

Comment:
Shouldn't we wrap to try catch JSON.parse? seems like a weak place

Reply:
done